### PR TITLE
feat(editor): update expressions display

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/components/ParameterInputWrapper.vue [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		methods: {
+			onFocus() {
+				this.$emit('focus');
+			},
+			onBlur() {
+				this.$emit('blur');
+			},
+			onDrop(data: string) {
+				this.$emit('drop', data);
+			},
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/ParameterInputWrapper.vue:186]: Would be nice to send the `$event` itself as well with the re-emits.

**Review by alexgrozav — DISMISSED** [review]
Awesome work, @mutdmour!

**Review by alexgrozav — APPROVED** [review]
Works beautifully. Good job, @mutdmour!

**Comment by janober** [comment]
Got released with `n8n@0.198.0`

**Pull request merged**